scrowl-io/scrowl
# Feature/sc 1332 theming alignment

### Short summary

- This imports the global bootstrap theme variables into the project to allow their use inside Scrowl
- remove variables and styles that were moved to owlui
- Make the bootup size wider
- Make Theme named `Default` instead of `Light`
- Remove unneeded `_reset.scss`
- Refactor main app layout 
- Add `href` prop to Logo and allow non-link usage
- Change some Variables in `pane`
- Inherit more from bootstrap in the Toolbar scss
- remove `class="light"` from main html 
- Inherit Tabs styling from theme, remove alterations in editor pane details.
- Refactor Home Page layout and styles
- Refactor Settings Page Layout and styles, use Tabs instead of Routes, use Radio instead of toggle button
- Change advanced settings to have unique content from theme settings.
- Make a `patterns` mixin file 
---

### Test steps

1. Import into stylesheet and attempt to use a mixin or sass variable from owlui
2. On build the owl logo should appear coloured correctly.

![image](https://user-images.githubusercontent.com/4621427/187983046-130ee578-724f-4782-9476-04e68dd6fd10.png)

![image](https://user-images.githubusercontent.com/4621427/187986262-51dddd7f-e878-4711-8212-41b8b4103581.png)
![image](https://user-images.githubusercontent.com/4621427/187986284-4a575a59-c182-4c88-aa1f-025b633d1014.png)

![image](https://user-images.githubusercontent.com/4621427/187982936-0bc72f0b-4e7a-48af-9c58-c01154531240.png)
![image](https://user-images.githubusercontent.com/4621427/187982989-ef345f18-faa6-4cdb-86eb-f7406068a555.png)
